tests/integration/test_smoke.py

```diff
@@ -14,13 +14,13 @@
 
 TEST_SINGLE_THREAD_FILE = Path(__file__).parent / "single_thread_program.py"
 
-if sys.version_info < (3, 10):
+if sys.version_info < (3, 10):  # pragma: no cover
     STACK_METHODS = (StackMethod.SYMBOLS, StackMethod.BSS, StackMethod.HEAP)
     CORE_STACK_METHODS = (StackMethod.SYMBOLS, StackMethod.BSS)
-elif sys.version_info < (3, 11):
+elif sys.version_info < (3, 11):  # pragma: no cover
     STACK_METHODS = (StackMethod.SYMBOLS, StackMethod.ELF_DATA, StackMethod.HEAP)
     CORE_STACK_METHODS = (StackMethod.SYMBOLS, StackMethod.ELF_DATA)
-else:
+else:  # pragma: no cover
     STACK_METHODS = (StackMethod.SYMBOLS, StackMethod.ELF_DATA)
     CORE_STACK_METHODS = (StackMethod.SYMBOLS, StackMethod.ELF_DATA)
```

Comment:
One of the three branches will be covered, but we don't know which, because the user could do `make check PYTHON=python3.9` for instance, so it makes sense to ignore the whole conditional for the purposes of branch checking.

Reply:
Ah I see.